k8s_itests/Makefile

```diff
@@ -21,18 +21,24 @@ endif
 
 all: clean itest
 
-fake_cluster: clean create_cluster
+fake_cluster: create_cluster
+ifeq (,$(wildcard ./.fake_cluster))
```

Comment:
i *think* we might be able to just native make targets to do this (e.g., have a .fake_cluster target that does the work in this conditional - but we might want someone better at make to chime in here (e.g., @EvanKrall)